specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -276,7 +277,7 @@
         ],
         "summary": "Returns the status for a specific document",
         "description": "Returns the translation status for a specific document based on the request Id and document Id.",
-        "operationId": "DocumentTranslation_GetDocumentStatus",
+        "operationId": "DocumentTranslation_GetDocument",
```

Comment:
I advise leaving `Status` on it. Otherwise, it gives the idea that the user can get the Document per se when in reality, the user gets information about the document.

Reply:
That's a good point!

@lmazuel, @KrzysztofCwalina - Do you have any thoughts?

Reply:
How about GetTranslations and GetDocuments?  both are LIST operations on the statuses

Reply:
I think it's better to have GetDocumentStatus. GetTranslations/GetDocuments are OK, but I agree with Mariana that it might be confusing to some (or in the future if we add a method to actually download the document).

Reply:
I still think we should have status in the list calls

Reply:
I'm on the fence about `GetTranslationStatus` vs `GetTranslation`. `GetTranslation` may be clear enough that it is the translation operation that is being retrieved. For the operations with Document in it I agree it may be confusing and users may think they are getting the document or documents.

This is the operation renames currently in the PR
```
CancelOperation => CancelTranslation
GetDocumentFormats => GetSupportedDocumentFormats
GetDocumentStorageSource => GetSupportedStorageSources
GetGlossaryFormats => GetSupportedGlossaryFormats
GetOperationDocumentsStatus => GetDocumentsStatus
GetOperations => GetTranslationsStatus
GetOperationStatus => GetTranslationStatus
SubmitBatchRequest => StartTranslation
```

I'd like to get this PR completed today. @alexhcheng, @KrzysztofCwalina do we agree on the list above?

/cc: @lmazuel